pkgs/development/python-modules/python-gnupg/default.nix

```
@@ -9,8 +9,13 @@ buildPythonPackage rec {
     sha256 = "3aa0884b3bd414652c2385b9df39e7b87272c2eca1b8fcc3089bc9e58652019a";
   };
 
+  patches = [
+    # TODO: Remove when https://bitbucket.org/vinay.sajip/python-gnupg/issues/143/pr-fix-tests-when-etc-foo-is-writable lands.
+    ./fix_permission_denied_test.patch
```

so that the package can still be updated by bot
```suggestion
  ] ++ stdenv.lib.optionals (lib.versionOlder version "0.4.7") [
    ./fix_permission_denied_test.patch
  ];
```